<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="17" column="4" colspan="2">
      <widget class="QPushButton" name="buttonAnnuler">
       <property name="text">
        <string>annuler</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="buttonPOW">
       <property name="text">
        <string>POW</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>                 </string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="buttonMEAN">
       <property name="text">
        <string>MEAN</string>
       </property>
      </widget>
     </item>
     <item row="11" column="5">
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="7">
      <widget class="QPushButton" name="buttonPlus">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="buttonTan">
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>    Rad/Deg:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="buttonCosh">
       <property name="text">
        <string>cosh</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="comOuiNon">
       <property name="text">
        <string>Non</string>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QPushButton" name="button9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="12" column="7">
      <widget class="QPushButton" name="buttonEVAL">
       <property name="text">
        <string>eval</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="buttonINV">
       <property name="text">
        <string>INV</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="buttonSQRT">
       <property name="text">
        <string>SQRT</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QPushButton" name="buttonFactoriel">
       <property name="text">
        <string>!</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QPushButton" name="buttonCUBE">
       <property name="text">
        <string>CUBE</string>
       </property>
      </widget>
     </item>
     <item row="12" column="5">
      <widget class="QPushButton" name="buttonPoint">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>    Expression :</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="buttonSin">
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="17" column="6" colspan="2">
      <widget class="QPushButton" name="buttonRetablir">
       <property name="text">
        <string>r√©tablir</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="buttonSQR">
       <property name="text">
        <string>SQR</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="buttonTanh">
       <property name="text">
        <string>tanh</string>
       </property>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="buttonDROP">
       <property name="text">
        <string>DROP</string>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QPushButton" name="button4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="10" column="5">
      <widget class="QPushButton" name="button5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="10" column="6">
      <widget class="QPushButton" name="button6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QPushButton" name="buttonDollar">
       <property name="text">
        <string>$</string>
       </property>
      </widget>
     </item>
     <item row="9" column="7">
      <widget class="QPushButton" name="buttonDim">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="14" column="4" colspan="2">
      <widget class="QPushButton" name="buttonSPACE">
       <property name="text">
        <string>SPACE</string>
       </property>
      </widget>
     </item>
     <item row="14" column="7">
      <widget class="QPushButton" name="buttonEntrer">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QPushButton" name="button7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>      Type:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QPushButton" name="buttonDUP">
       <property name="text">
        <string>DUP</string>
       </property>
      </widget>
     </item>
     <item row="12" column="6">
      <widget class="QPushButton" name="buttonExpression">
       <property name="text">
        <string>'</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="buttonLog">
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="buttonMOD">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QPushButton" name="buttonSWAP">
       <property name="text">
        <string>SWAP</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="buttonSUM">
       <property name="text">
        <string>SUM</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="button0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="14" column="6">
      <widget class="QPushButton" name="buttonCLEAR">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="buttonLn">
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="11" column="7">
      <widget class="QPushButton" name="buttonDiv">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="buttonSinh">
       <property name="text">
        <string>sinh</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mode">
       <property name="text">
        <string>entier</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="buttonSIGN">
       <property name="text">
        <string>SIGN</string>
       </property>
      </widget>
     </item>
     <item row="11" column="6">
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="RadouDeg">
       <property name="text">
        <string>rad</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="buttonCos">
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>  Complexe:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="7">
      <widget class="QPushButton" name="buttonMulti">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QPushButton" name="button8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>    Pile :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4" colspan="4">
      <widget class="QLineEdit" name="inputLine">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4" rowspan="4" colspan="2">
      <widget class="QTextEdit" name="Express">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="6" rowspan="4" colspan="2">
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuType_2">
    <property name="title">
     <string>type</string>
    </property>
    <widget class="QMenu" name="menuComplexe">
     <property name="title">
      <string>complexe</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionOui"/>
     <addaction name="actionNon"/>
    </widget>
    <addaction name="actionRad"/>
    <addaction name="actionDeg"/>
    <addaction name="separator"/>
    <addaction name="menuComplexe"/>
    <addaction name="separator"/>
    <addaction name="actionEnt"/>
    <addaction name="actionRationnel"/>
    <addaction name="actionRe"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuType_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionEntiere">
   <property name="text">
    <string>entiere</string>
   </property>
  </action>
  <action name="actionRationnelle">
   <property name="text">
    <string>rationnelle</string>
   </property>
  </action>
  <action name="actionReelle">
   <property name="text">
    <string>reelle</string>
   </property>
  </action>
  <action name="actionComplexe">
   <property name="text">
    <string>complexe</string>
   </property>
  </action>
  <action name="actionExpression">
   <property name="text">
    <string>expression</string>
   </property>
  </action>
  <action name="actionEntier">
   <property name="text">
    <string>entier</string>
   </property>
  </action>
  <action name="actionRad">
   <property name="text">
    <string>radiant</string>
   </property>
  </action>
  <action name="actionReel">
   <property name="text">
    <string>reel</string>
   </property>
  </action>
  <action name="actionDeg">
   <property name="text">
    <string>degre</string>
   </property>
  </action>
  <action name="actionOui">
   <property name="text">
    <string>oui</string>
   </property>
  </action>
  <action name="actionNon">
   <property name="text">
    <string>non</string>
   </property>
  </action>
  <action name="actionEnt">
   <property name="text">
    <string>entiere</string>
   </property>
  </action>
  <action name="actionRationnel">
   <property name="text">
    <string>rationnelle</string>
   </property>
  </action>
  <action name="actionRe">
   <property name="text">
    <string>reelle</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCLEAR</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
